src/internal/m365/service/groups/enabled.go

```diff
@@ -0,0 +1,26 @@
+package groups
+
+import (
+	"context"
+
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+)
+
+type getByIDer interface {
+	GetByID(ctx context.Context, identifier string) (models.Groupable, error)
+}
+
+func IsServiceEnabled(
+	ctx context.Context,
+	gbi getByIDer,
+	resource string,
+) (bool, error) {
+	_, err := gbi.GetByID(ctx, resource)
+	if err != nil {
+		// TODO(meain): check for error message in case groups are
+		// not enabled at all similar to sharepoint
+		return false, err
+	}
+
+	return true, nil
```

Comment:
We should already have called "getByID" at this point to get the resource's ID and Name, so I don't think we're gaining anything here by calling it again.
```suggestion
	return true, nil
```

Reply:
Sounds good. @meain is this okay with you?